### ISSUE TITLE
Feature req: provide an abstraction for 'relational database in lambda'

### ISSUE THREAD

**Issue by Birowsky** [punchcard/punchcard#120]
You've already done a beautiful job when providing dynamodb table as a lambda dependency.

Have you given this a thought for a relational database? Say, Aurora Serverless?

I've been trying to find a definitive answer for how to ideally set up rds for lambda without much success. So I thought there's a chance that you may've looked into this, you know, the regular serverless use case: Api Gateway -> Lambda -> RDS. I pointed out Aurora Serverless because of that 'data api' feature it provides.

Thanx!

**Comment by sam-goodwin** [comment]
Yeah i so keen to do this too! I've got a PR in the works that'll be tackling this problem exactyly - provisioning, schemas and migrations.

https://github.com/punchcard/punchcard/pull/112

Will be getting to that once i finish the AppSync stuff https://github.com/punchcard/punchcard/pull/119. Going to support calling from Lambda or from an AppSync resolver.

**Comment by Birowsky** [comment]
Awesome! Since it doesn't say anything about provisioning, do you mind sharing how do you intend to handle the db connection pool?
